src/AutoUI/index.tsx

```diff
@@ -69,6 +69,7 @@ import { NoRecordsFoundView } from './NoRecordsFoundView';
 const DEFAULT_ITEMS_PER_PAGE = 50;
 
 const HeaderGrid = styled(Flex)`
+	row-gap: 8px;
```

Comment:
Noting for later: We need to do the Flex/Box/Grid migration here as well, this looks like a spot that would use Grid